### PR TITLE
use command builtin for grepping

### PR DESCRIPTION
**Pull request by erikdstock**

### CODE REVIEW

**Comment by wilmoore** [comment]
Looks good.

What do you think about also doing this for `sort`.
What are your thoughts on also making those commands `local *` variables?

**Comment by erikdstock** [comment]
Sure thing- let me update it when i get home.

**Comment by erikdstock** [comment]
My first time doing locals in bash like this- i'm not sure it's the best approach.

File: php-version.sh [hunk 1]
```diff
@@ -17,14 +17,19 @@ function php-version {
   # PHP installation paths
   local _PHP_VERSIONS=""
 
+  # use builtin commands for sort and grep
+  local grep="command grep -E"
+  local sort="command sort -r -t . -k 1,1n -k 2,2n -k 3,3n"
+
+
```
Comment (wilmoore) [inline php-version.sh]: Mind dropping the extra line-break here?

**Comment by wilmoore** [comment]
Looks great @erikdstock - I just had one comment. If you could address that, I'd be happy to merge.

**Comment by erikdstock** [comment]
Done- thanks!

**Comment by wilmoore** [comment]
LGTM 👍

**Pull request merged**